src/GuestConfiguration/GuestConfiguration/Cmdlets/GetAzVMGuestPolicyStatus.cs

```diff
@@ -20,14 +20,15 @@ namespace Microsoft.Azure.Commands.GuestConfiguration.Cmdlets
     using Microsoft.Azure.Commands.GuestConfiguration.Common;
     using Microsoft.Azure.Commands.GuestConfiguration.Models;
     using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
-    using Microsoft.Azure.Management.GuestConfiguration.Models;
+    using Microsoft.Azure.Commands.GuestConfiguration.Helpers;
 
     /// <summary>
     /// Gets Vm Guest Policy reports (GuestConfiguration policy reports)
     /// </summary>
-    [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "VMGuestPolicyReport", DefaultParameterSetName = ParameterSetNames.VmNameScope), 
-        OutputType(typeof(IList<GuestConfigurationAssignmentReport>))]
-    public class GetAzureRmVMGuestPolicyReport : GuestConfigurationCmdletBase
+    [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzurePrefix + "VMGuestPolicyStatus", DefaultParameterSetName = ParameterSetNames.VmNameScope)]
+    [OutputType(typeof(PolicyStatusDetailed))]
+    [OutputType(typeof(PolicyStatusDetailed[]))]
```

Comment:
do not specify the collection - powershell treats single values and collections alike, your first outputtype declaration is sufficient.

Reply:
done